content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,38 @@
+---
+title: IMDSv2 Enablement By Default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+# IMDSv2 Default Behavior and Host Metadata Enhancement
+
+Agent v7.64.0 enables IMDSv2 by default for enhanced security and improved metadata handling. The host metadata payload contains a field, `legacy-resolution-hostname`, that captures the AWS instance hostname using legacy resolution logic (when IMDSv2 was disabled) and is only included when needed.
+
+The hostname displayed in Datadog **can** change to show the instance ID as the hostname. Datadog continues to use the original hostname to tag the different metrics sent by the Agent.
+
+## Field inclusion criteria
+
+The `legacy-resolution-hostname` field appears in the host metadata payload only when **both** of the following conditions are met:
+
+- The configuration flag `ec2_imdsv2_transition_payload_enabled` is set to `true`.
+- The legacy hostname retrieved is different from the hostname retrieved with IMDSv2.
+
+## Impact on hostname reporting
+
+The addition of the `legacy-resolution-hostname` field **does not** alter the hostname that is used to tag the metrics sent by the Agent. If the Agent previously did not use the instance ID as the hostname, it continues to use the original hostname for metric tagging. This ensures continuity and avoids any disruption in monitoring or alerting setups.
+
+The displayed hostname **can** switch to the instance ID in Fleet Automation and the Infrastructure List. However, you can still locate your host using the original hostname.
+
+## Example metadata payload
```

Comment:
Mentioned earlier but as we go through the doc, there's a focus on developer persona detail level. We should rework this doc to focus on higher-level customer-visible effects and things to look out for. 

Some examples:
- `After upgrading to v<VER> in <CONDITIONs>, you may see <X, Y, Z>. We expect <A,B,C> to be impacted by this. If you see problems you can continue using the previous behavior by <PROCEDURE>`
- `You may notice a delay in your <FEATURE(S)> for X amount of time ...`
- `If you already had the flag <FLAG>, your machine has already been reporting that way and will see no change`